site/content/en/guides/consumer/compositepkgs/_index.md

```diff
@@ -0,0 +1,186 @@
+---
+title: "Get and apply a remote composite package"
+linkTitle: "Composite Package"
+weight: 7
+type: docs
+description: >
+  Fetch a composite package from a remote git repository and apply its contents to
+  a cluster
+---
+
+{{% pageinfo color="warning" %}}
+
+#### Notice: Composite packages feature support is in alpha phase
+
+{{% /pageinfo %}}
+
+This guide walks you through an example to get, view, set and apply contents of a
+kpt package which has nested subpackages in it. A kpt package is a directory of
+resource configs with a valid `Kptfile` in it. A composite package is a `kpt` package
+with 1 or more subpackages in its directory tree.
+
+Principles:
+
+1. Each kpt package is an independent building block and
+   should contain resources(ex: setter definitions) of its own.
+2. If a package is present in the directory tree of parent package,
+   the configs of that package are out of scope for the actions performed
+   on the parent package.
+3. `--recurse-subpackages(-R)` flag can be leveraged to run a command
```

Comment:
Doesn't 3 negate 2? Or is 2 describing the default behavior where -R is not provided? Make this clear

Reply:
Rephrased the sentence.